recipes/so3/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+from glob import glob
+import os
+
+
+
+class So3Conan(ConanFile):
+    name = "so3"
+    license = "GPL-3.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/astro-informatics/so3"
+    description = "Fast and accurate Wigner transforms"
+    settings = "os", "arch", "compiler", "build_type"
+    topics = ("physics", "astrophysics", "radio interferometry")
+    options = {"fPIC": [True, False]}
```

Comment:
On mobile so I couldn't  tell..

Is shared not possible?

Reply:
This package depends on #4820 which does not support shared at this juncture.
I might put it in at some point, but I first want to move things from bintray.